frontend/src/pages/Help.tsx

```diff
@@ -0,0 +1,187 @@
+import "../index.css";
+import React from "react";
+import { AuthContext } from "../components/AuthProvider";
+import { HeaderUser } from "../components/HeaderUser";
+import { Row } from "../components/Row";
+import { Cell } from "../components/Cell";
+import { IssueActivityPair, FetchedTimeEntry } from "../model";
+import { QuickAdd } from "../components/QuickAdd";
+import weektravel from "../icons/weektravel.png";
+import overview from "../icons/overview.png";
+
+export const Help = () => {
+  const context = React.useContext(AuthContext);
+  const exampleIAP: IssueActivityPair = {
+    activity: {
+      id: 1,
+      name: "Test activity",
+    },
+    issue: {
+      id: 1,
+      subject: "Test issue",
+    },
+    custom_name: "Test custom name",
+    is_hidden: false,
+  };
+  const exampleEntry: FetchedTimeEntry = {
+    id: 1,
+    project: {
+      id: 1,
+      name: "Test project",
+    },
+    issue: {
+      id: 1,
+    },
+    user: {
+      id: 1,
+      name: "Test user",
+    },
+    activity: {
+      id: 1,
+      name: "Test activity",
+    },
+    hours: 1,
+    comments: "Test comments",
+    spent_on: "2020-01-01",
+    created_on: "2020-01-01",
+    updated_on: "2020-01-01",
+  };
+  return (
+    <main>
+      <div className="usr-header">
+        <h1 className="help-title">How do I use the urdr service?</h1>
+        <HeaderUser username={context.user ? context.user.login : ""} />
+      </div>
+      <div className="help-wrapper">
+        <h2 className="help-subtitle">Introduction</h2>
+        <p className="help-info">
+          The purpose of this website is to ease the process of logging time on
+          the
+          <a href="https://projects.nbis.se"> NBIS Redmine instance</a>. The
+          usual procedure for logging time with Redmine involves selecting an
+          issue and an activity for reporting your time. In urdr, issues are
+          strictly linked to valid activities, thus making it simpler to do time
+          reporting accurately. In order to improve the user experience, the
+          time reporting page offers a spreadsheet-like grid for time entry
+          reporting, together with an intuitive way of navigating over different
+          weeks. The report view is primarily made up of a header containing a
+          week navigation handler and a user dropdown menu, the time entries
+          grid, and the bottom bar including the feature for adding new rows and
+          saving your changes.
```

Comment:
To me it sounds like we are bragging about "improving the user experience" and "intuitive ways" :) If I would visit this page as a user, it would be because I'm confused about something. Then I would rather have a neutral explanation and not having the text tell me how nice the page is, it would make me feel dumb. 
```suggestion
          <a href="https://projects.nbis.se"> NBIS Redmine instance</a>. The
          usual procedure for logging time with Redmine involves selecting an
          issue and an activity for reporting your time. In urdr, issues are
          strictly linked to valid activities, thus making it simpler to do time
          reporting accurately. The time reporting page offers a spreadsheet-like grid for time entry
          reporting, together with the possibility to navigate over different
          weeks. The report view is primarily made up of a header containing a
          date picker and arrow buttons to navigate between weeks, as well as a user dropdown menu.
          Below, you find the time entries grid, and the bottom bar including the feature for adding new rows and
          saving your changes.
```